docs/api-table.js

```diff
@@ -1373,6 +1373,7 @@ export const elements = {
       ['type', "'success' | 'warning' | 'error'", "'success'", 'Type of toast'],
       ['text', 'string', 'undefined', 'Applies styling to indicate click-ability'],
       ['duration', 'number', '5000', 'Auto removal of toast'],
+      ['canclose', 'boolean', 'false', 'Adds close button'],
```

Comment:
Wdyt about adding the following text here 👇 It is from the toast-container and clearly explains that this is not we recommend to use.
`Whether the toast can be dismissed. Defaults to false. WARNING! For accessibility reasons, toasts should not be interactive and canclose should always be false. If the toast absolutely must be dismissible, set this to true.`
And maybe also add the same [here](https://github.com/warp-ds/tech-docs/pull/78/files#diff-a443f2b0c15bff5190ab54eaf713e90eef895ef8d557e80b1427d93b30c95bdeR61)

Reply:
Aa okay! I added a similar text myself in the in the visual options. But we can add it here also 👍